src/rest-server/docs/swagger.yaml

```diff
@@ -1533,22 +1533,11 @@ components:
           properties:
             submitFrom:
               type: string
-            hivedscheduler:
+            hivedScheduler:
               type: object
               properties:
                 jobPriorityClass:
                   type: string
-                taskRoles:
-                  type: object
-                  additionalProperties:
-                    type: object
-                    properties:
-                      gpuType:
-                        type: string
-                      reservationId:
-                        type: string
-                      affinityGroupName:
-                        type: string
```

Comment:
could you only hide reservationId and affinityGroupName? I think we should rename gpuType to skuType and keep it here.

Reply:
updated